lib/tasks/fixes.rake

```diff
@@ -1,4 +1,16 @@
 namespace :fixes do
+  desc "Find mismatched LAA references"
+  task find_mismatched_references: :environment do
+    submissions_to_check = Submission.where("application.current_version > 1")
```

Comment:
my thinking here was that we might catch some crazy edge cases that we otherwise missed but yeah - nothing seems possible in that regard so will update to 2